docs/devel/release-process.rst

```diff
@@ -27,43 +27,35 @@ Process
 
 The process for cutting a minor release is as follows:
 
-#. Ensure upgrading document exists in docs/admin/upgrading
+#. Ensure upgrading document exists in docs/tasks/upgrading
 
+#. Ensure all strings of versions have been updated:
+
+   * deploy/charts/cert-manager/README.md
+   * docs/getting-started/install/kubernetes.rst
+   * docs/getting-started/install/openshift.rst
+   * docs/getting-started/webhook.rst
+   * docs/tutorials/acme/quick-start/index.rst
 #. Create a new release branch (e.g. ```release-0.5```)
 
 #. Push it to the ```jetstack/cert-manager`` repository
 
 #. Create a pull-request updating the Helm chart version and merge it:
 
-   * Update contrib/charts/cert-manager/README.md
-   * Update contrib/charts/cert-manager/Chart.yaml
-   * Update contrib/charts/cert-manager/values.yaml
-   * Update contrib/charts/cert-manager/requirements.yaml
-   * Update contrib/charts/cert-manager/webhook/Chart.yaml
-   * Update contrib/charts/cert-manager/webhook/values.yaml
-   * Run ```helm dep update``` in the contrib/charts/cert-manager directory
-   * Run ```./hack/update-deploy-gen.sh``` in the root of the repository
+   * Run ```helm dep update``` in the deploy/charts/cert-manager directory
```

Comment:
This shouldn't be needed? The actual building of the chart is handled by Prow/our release pipeline 😄

Reply:
whoops